src/components/EmptyState/EmptyState.scss

```diff
@@ -1,6 +1,6 @@
 @import '../../styles/common';
 
-$detail-width: rem(336px);
+$detail-width: rem(400px);
```

Comment:
I think we may only want to make this change to in content empty states and centered layout empty states. Looks like it could cause problems with legacy layouts

![image](https://screenshot.click/Build_13871_-_Shopify_-_Percy__Visual_testing_as_a_service_2020-06-09_09-47-21.png)

Reply:
This shouldn't break current empty states, as `$detail-width` is just the width of the content left of the illustration (so the illustration might shift a bit, but the illustration itself get's pull left using negative margin so it might not actually be affected at all 🤔). Happy to tophat in `web` to help double check 👍